test/integration/test_global_catalog_v1.py

```diff
@@ -41,7 +32,9 @@ class TestGlobalCatalogV1(unittest.TestCase):
     """
     @classmethod
     def setUpClass(self):
-        if not configLoaded:
+        if os.path.exists(config_file):
+            os.environ['IBM_CREDENTIALS_FILE'] = config_file
+        else:
             raise unittest.SkipTest(
```

Comment:
@padamstx I don't think we need to load the config file to a config variable with the `read_external_sources` function since we are not using it in this file. What is your take on this? Is this okay or we should load it anyway.

Reply:
You're right, if we're not retrieving any test-specific config properties (e.g. account id, etc.) then no need to explicitly call read_external_sources to retrieve the config dictionary.